contracts/test/EVMPrecompileTester.js

```diff
@@ -0,0 +1,190 @@
+    
+                // Get a contract instance
+                erc20 = new ethers.Contract(contractAddress, contractABI, signer);
+                console.log("end of before");
+            });
+    
+            it("Transfer function", async function() {
+                const receiver = '0x70997970C51812dc3A010C7d01b50e0d17dc79C8';
+                const beforeBalance = await erc20.balanceOf(owner);
+                const tx = await erc20.transfer(receiver, 1);
+                const receipt = await tx.wait();
+                expect(receipt.status).to.equal(1);
+                const afterBalance = await erc20.balanceOf(owner);
+                const diff = beforeBalance - afterBalance;
+                expect(diff).to.equal(1);
+            });
+    
+            it("Approve and TransferFrom functions", async function() {
```

Comment:
can you do a transferFrom without doing an approve and make sure it fails?

Reply:
Updated